recipes/entrypoints/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.2" %}
+
+package:
+    name: entrypoints
+    version: {{ version }}
+
+source:
+    fn: {{ version }}.tar.gz
+    url: https://github.com/takluyver/entrypoints/archive/{{ version }}.tar.gz
+    md5: bcbad7a52241a7e604657675f94f2f08
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - configparser # [py2k]
+    run:
+        - python
+
+test:
+    imports:
+        - entrypoints
+
+about:
+    home: http://entrypoints.readthedocs.org/en/latest/
+    license: MIT
+    summary: Discover and load entry points from installed packages
+
+extra:
+  recipe-maintainers:
+    - pelson
+    - minrk
+    - takluyver
```

Comment:
@takluyver are you OK being a maintainer of your package here at conda-forge?
@minrk since you are already a maintainer of `nbconvert` I also added you here. Is that OK?